googet.goospec

```diff
@@ -1,6 +1,6 @@
 {
   "name": "googet",
-  "version": "2.14.1@1",
+  "version": "2.14.1@2",
```

Comment:
Sorry should have been more specific, thats the build number. Should be 2.14.2@1